component/credentialstatus/credentialstatus_service.go

```diff
@@ -153,6 +153,7 @@ func (s *Service) UpdateVCStatus(profileID profileapi.ID, vcID, vcStatus string,
 		KMS:                     keyManager,
 		SignatureRepresentation: issuerProfile.VCConfig.SignatureRepresentation,
 		VCStatusListType:        issuerProfile.VCConfig.Status.Type,
+		SDJWT:                   issuerProfile.VCConfig.SDJWT,
```

Comment:
what is the purpose of this property?

Reply:
this is the profile configuration struct that holds SD-JWT params. Pls see: https://github.com/trustbloc/vcs/pull/1041/files#diff-5c624848ef5de19a8689da636c2e03bfa9761a9da5b6b298c09420691684be1eR13

Reply:
I don't think it is actually needed, lets create a follow up & discuss this